src/apiCalls/index.js

```diff
@@ -2,3 +2,6 @@ export { getFromCart } from './cart/getFromCart'
 export { addToCart } from './cart/addToCart'
 export { incDescQuantity } from './cart/incDescQuantity'
 export { removeFromCart } from './cart/removeFromCart'
+export { getFromWishlist } from './wishlist/getFromWishlist'
+export { addToWishlist } from './wishlist/addToWishlist'
+export { removeFromWishlist } from './wishlist/removeFromWishlist'

```

Comment:
You can have all the above functions in a single file named helper functions or something like that as you'll also require them and few more for carts page functionality.

Reply:
Okay noted.